pkgs/applications/window-managers/i3/status-rust.nix

```diff
@@ -2,16 +2,16 @@
 
 rustPlatform.buildRustPackage rec {
   pname = "i3status-rust";
-  version = "0.14.1";
+  version = "0.14.2";
 
   src = fetchFromGitHub {
     owner = "greshake";
     repo = pname;
     rev = "v${version}";
-    sha256 = "11qhzjml04njhfa033v98m4yd522zj91s6ffvrm0m6sk7m0wyjsc";
```

Comment:
I prefer this one because it can be easier copied with one click and has better syntax highlighting in vim. Also nix-build shows me that if it is wrong.

Reply:
I agree, have switched it over to using the base32 version

Reply:
FYI in the long term it's planned to switch to those SRI hashes and they are already provided by default on e.g. `nixUnstable`. However those are interchangeable atm and it's fine to use whatever format Nix suggests, I just don't think that it's relevant to make such a change request part of a review :) 

That said, I'm currently building/testing it locally and will merge if everything seems fine.